builder/steps/gen-dockerfile/contents/data/Dockerfile.txt

```diff
@@ -25,7 +59,9 @@ RUN yarn global add npm@'<%- config.npmVersion %>'
 RUN yarn global add yarn@'<%- config.yarnVersion %>'
 <% } %>
 
+<% if (!config.hasBuildCommand) { %>
```

Comment:
As discussed, this if can be avoided if you move the COPY up to the else block above.

Reply:
I've moved it.